ui/app/controllers/allocations/allocation/index.js

```diff
@@ -47,10 +48,22 @@ export default class IndexController extends Controller.extend(Sortable) {
   }
 
   @computed('model.taskGroup.services.@each.name')
-  get services() {
+  get groupServices() {
     return (this.get('model.taskGroup.services') || []).sortBy('name');
```

Comment:
`suggestion:` This is an asynchronous request. `allocations.allocation` does not load the `TaskGroup` and this should trigger a fetch request. `TaskGroup` can fail as a request here and we won't notify the user.